api/database/factories/ApplicantFilterFactory.php

```diff
@@ -43,6 +44,13 @@ public function definition()
                 array_column(WorkRegion::cases(), 'name'),
                 $this->faker->numberBetween(1, 3)
             ),
+            'flexible_work_locations' => array_merge(
+                [FlexibleWorkLocation::ONSITE->name],
+                $this->faker->randomElements(
+                    [FlexibleWorkLocation::REMOTE->name, FlexibleWorkLocation::HYBRID->name],
+                    $this->faker->numberBetween(0, 2)
+                )
+            ),
```

Comment:
Safe to assume the field in practice will be ONSITE plus 0-2 of the flex options